doc/_includes/header.html

```diff
@@ -41,18 +41,18 @@
 
       <section id="menu">
         <ul>
-          <li><a href="{{ site.baseurl }}index.html">Introduction</a></li>
-          <li><a href="{{ site.baseurl }}news/">News</a></li>
-          <li><a href="{{ site.baseurl }}install.html">Installation</a></li>
-          <li><a href="{{ site.baseurl }}language.html">Schema Language</a></li>
-          <li><a href="{{ site.baseurl }}encoding.html">Encoding</a></li>
-          <li><a href="{{ site.baseurl }}rpc.html">RPC Protocol</a></li>
-          <li><a href="{{ site.baseurl }}capnp-tool.html">The <code>capnp</code> Tool</a></li>
-          <li><a href="{{ site.baseurl }}cxx.html">C++ Serialization</a></li>
-          <li><a href="{{ site.baseurl }}cxxrpc.html">C++ RPC</a></li>
-          <li><a href="{{ site.baseurl }}otherlang.html">Other Languages</a></li>
-          <li><a href="{{ site.baseurl }}roadmap.html">Road Map</a></li>
-          <li><a href="{{ site.baseurl }}faq.html">FAQ</a></li>
+          <li><a href="{{ site.baseurl }}index.html"><p>Introduction</p></a></li>
```

Comment:
The main point of the `<p>` was to allow the links to be sized. In #269 I instead use `display:block` to achieve the same thing.